drivers/ever-hid.c

```diff
@@ -210,18 +210,21 @@ static const char* ever_messages_fun(double value)
 
 	int n = 0;	/* number of characters currently in line */
 
+	/* duplicate of ups.status: OB LB*/
```

Comment:
I think we have a style documented somewhere for having whitespace between starting tag, content and ending tag of the comments (here and in numerous "not implemented" comments below, the space is missing before `*/` ending)